### ISSUE TITLE
host/maipo: drop tuned

### ISSUE THREAD

**Issue by lucab** [openshift/os#265]
We are currently shipping [tuned](https://github.com/openshift/os/blob/c29249b5307edcd1b7317202edcde92138470be6/host-maipo.yaml#L26) (with its web of python dependencies) to effectively apply a bunch of [sysctl values](https://github.com/redhat-performance/tuned/blob/master/profiles/atomic-host/tuned.conf).

I pretty much agree with the `(legacy IMO)` comment there, as we already have [systemd-sysctl](https://www.freedesktop.org/software/systemd/man/sysctl.d.html) in the OS.

We can directly convert those settings into a vendor conf-file and drop tuned. This will help in both our efforts of de-bloating the OS in general, and removing python bits in particular.

Open questions I have:
 * is the tuned-to-sysctld effort already ongoing somewhere else? (I didn't find any)
 * where (as in, repo+srpm+rpm location) can we shove these rhcos vendor files?
 * is there anything else I missed?

**Comment by cgwalters** [comment]
old tuned discussion: https://lists.projectatomic.io/projectatomic-archives/atomic-devel/2015-May/msg00039.html

Slightly newer: https://lists.projectatomic.io/projectatomic-archives/atomic-devel/2016-October/msg00005.html

I think the most logical place for our set of sysctls is in [the release package](https://github.com/openshift/redhat-release-coreos).

**Comment by ashcrow** [comment]
> I think the most logical place for our set of sysctls is in the release package.

:+1:

**Comment by lucab** [comment]
@jeremyeder I did, thanks. And I'm also very happy about tuned-operator. And I also followed-up with the presenter to ask about systemd-sysctl.d integration. This ticket is in the specific context of the current rhcos-maipo image and immutable provisioning of the OS on first-boot. I don't want to marginalize any technology and I don't doubt that that fits pretty well in other usecases, sorry if that was the nuance of my words that came across (unfortunately this is not my native language).

On a more technical note, if there are other (non-sysctl) settings that we are missing after removing this, we should take a followup ticket and investigate how to provision those on first-boot before the pivot-root happens.
